data/pathfinder/paizo/roleplaying_game/core_rulebook/cr_equipmods.lst

```diff
@@ -295,63 +295,63 @@ Wounding			SORTKEY:+2	KEY:Special Ability ~ Wounding ~ Amulet of Mighty Fists		F
+Intelligent Magic Item Base																KEY:Intelligent Item ~ Base							NAMEOPT:TEXT=Intelligent	TYPE:Weapon.Armor.Goods	COST:500	VISIBLE:QUALIFY																																																																																			PRETYPE:1,ArmorEnhancement,WeaponEnhancement																												BONUS:VAR|IntItemSenseRange|30	BONUS:VAR|IntItemNegativeLevel|1|PREVARGTEQ:IntelligentItemEgo,20	BONUS:VAR|IntItemNegativeLevel|1|PREVARGTEQ:IntelligentItemEgo,30	BONUS:VAR|NegativeLevel|1|PREVARLTEQ:IntelligentItemEgo,19	BONUS:VAR|NegativeLevel|1|PREVARLTEQ:IntelligentItemEgo,29	BONUS:VAR|NegativeLevel|1|PREVARGTEQ:IntelligentItemEgo,30	BONUS:VAR|IntItemStatINT|10	BONUS:VAR|IntItemStatWIS|10	BONUS:VAR|IntItemStatCHA|10	BONUS:VAR|BaseCostTracker|COST	BONUS:VAR|IntelligentItemEgo|(BaseCostTracker>=1001)+(BaseCostTracker>=5001)+(BaseCostTracker>=10001)+(BaseCostTracker>=20001)+(BaseCostTracker>=50001)+(BaseCostTracker>=50001)+(BaseCostTracker>=100001)+(BaseCostTracker>=100001)+(BaseCostTracker>=200001)+(BaseCostTracker>=200001)+(BaseCostTracker>=200001)+(BaseCostTracker>=200001)	SPROP:Intelligence %, Wisdom %, Charisma %, Ego Score %|IntItemStatINT|IntItemStatWIS|IntItemStatCHA|IntelligentItemEgo	SPROP:Empathy allows the item to encourage or discourage certain actions by communicating emotions and urges. It does not allow for verbal communication.|!PREEQUIP:1,EQMOD=Intelligent Item ~ Communication / Speech,EQMOD=Intelligent Item ~ Communication / Telepathy	ABILITY:Intelligent Item|AUTOMATIC|Base Intelligent Item											DEFINE:IntItemSenseRange|0	DEFINE:SpeechBonusLang|0	DEFINE:IntItemNegativeLevel|0	DEFINE:IntelligentItemEgo|0	DEFINE:IntItemStatINT|0	DEFINE:IntItemStatWIS|0	DEFINE:IntItemStatCHA|0	DEFINE:BaseCostTracker|0
```

Comment:
Intelligent items can be other items than weapons and armor. This may need to be fixed in a different manner, or remove the PRETYPE completely.
From the core rulebook: "Only permanent magic items (as opposed to single-use items or those with charges) can be intelligent. (This means that potions, scrolls, and wands, among other items, are never intelligent.)"

Reply:
Do intelligent items require a +1 enhancement be made before becoming intelligent?  I was not sure about that.

Reply:
It only requires that the item be magic. Since non-weapon, non-armor items can also be intelligent items, and they don't have +1 or higher bonuses at all, something else needs to be used here. Maybe PRETYPE:1,Magic

Reply:
Actually iirc (at least in 3.5) the only requirement for a item to become intelligent is actually has to be worth a certain amount of coin

Reply:
LLisandur is correct.  Actual text: 
 Only permanent magic items (as opposed to single-use items or those with charges) can be intelligent. (This means that potions, scrolls, and wands, among other items, are never intelligent.) In general, less than 1% of magic items have intelligence.